.github/workflows/release.yaml

```diff
@@ -28,8 +28,8 @@ jobs:
       uses: docker/login-action@v3
       with:
         registry: ${{ vars.PUBLIC_REGISTRY }}
-        username: ${{ secrets.PUBLIC_REGISTRY_USERNAME }}
-        password: ${{ secrets.PUBLIC_REGISTRY_PASSWORD }}
+        username: ${{ secrets.docker_username }}
+        password: ${{ secrets.docker_password }}
```

Comment:
I think these secrets are not yet available in the repository settings, are they? Just mentioning so that we make sure that they are set properly.